dateutil/parser/_parser.py

```diff
@@ -1091,6 +1095,12 @@ def _parse_min_sec(self, value):
         # TODO: Every usage of this function sets res.second to the return
         # value. Are there any cases where second will be returned as None and
         # we *dont* want to set res.second = None?
+
+        # See GH issue #427, fixing float rounding
+        # Conversion here might be redundant as value is already passed as Decimal()
+        # but _parse_min_sec() is also called in _parse_numeric_token where it is still a float().
```

Comment:
If it's already `float`, you're not making anything better by converting it to `Decimal`.

```python
>>> from decimal import Decimal
>>> Decimal(5.6)
Decimal('5.5999999999999996447286321199499070644378662109375')
```

The problem is to avoid converting it to a `float` in the first place.

Reply:
If this is not causing any problems just drop the `Decimal` call.